src/module/comment_display_generation.js

```diff
@@ -56,38 +57,17 @@ const generateInfo = (data) => {
   return details;
 };
 
-const generateComments = () => {
+const generateComments = (mealId) => {
   const comments = document.createElement('div');
-  comments.classList.add('comments');
-  comments.classList.add('flex-column');
-  commentList.forEach((commentItem) => {
-    const commentCard = document.createElement('div');
-    commentCard.classList.add('comment-card');
-    commentCard.classList.add('flex-column');
-    const commentHeader = document.createElement('div');
-    const date = document.createElement('span');
-    const name = document.createElement('span');
-    const message = document.createElement('div');
-
-    date.classList.add('date');
-    date.textContent = commentItem.creation_date;
-
-    name.classList.add('name');
-    name.textContent = ` | By ${commentItem.username}`;
-
-    commentHeader.classList.add('comment-header');
-    commentHeader.appendChild(date);
-    commentHeader.appendChild(name);
-
-    message.classList.add('message');
-    message.textContent = commentItem.comment;
-
-    commentCard.appendChild(commentHeader);
-    commentCard.appendChild(message);
-
-    comments.appendChild(commentCard);
-  });
-
+  comments.classList.add('comments', 'flex-column');
+  const commentTitle = document.createElement('h2');
+  commentTitle.classList.add('sub-title', 'meal-title');
+  getComment(mealId)
+    .then((response) => {
+      const commentNumber = commentCounter(response);
+      commentTitle.innerHTML = `Comments [${commentNumber}]`;
```

Comment:
- [ ] 
**When Adding A comment, the comment Number will not increase. You can try playing around with your code to update the comment live or you already have a set timeout function you can just add ``` window.location.reload() ``` to save yourself some time**

![galti2](https://user-images.githubusercontent.com/104627775/187952369-e556e745-08f7-4018-87b1-a875e0c302bb.PNG)

Reply:
Yes, I totally missed that! :smiley: 
I just fixed that!
Thanks!